captum/insights/api.py

```diff
@@ -502,15 +502,17 @@ def _get_outputs(self) -> List[Tuple[VisualizationOutput, SampleCache]]:
         batch_data = next(self._dataset_iter)
         vis_outputs = []
 
-        for inputs, additional_forward_args, label in _batched_generator(
+        # Type ignore for issue with passing union to function taking generic
+        # https://github.com/python/mypy/issues/1533
+        for inputs, additional_args, label in _batched_generator(  # type: ignore
```

Comment:
These changes were necessary to fix mypy failures on master due to merging with the common type hints. additional_forward_args was renamed to additional_args since the "type: ignore" made the line too long causing flake8 errors.
CC: @edward-io

Reply:
could we do:

```
for (
  inputs,
  additional_forward_args,
  label,
) in _batched_generator( # type: ignore
```

instead?

Reply:
Sure, I figured a single line is more concise, but this works too, will update, thanks!